contact.html

```diff
@@ -139,56 +141,53 @@
         <div class="row">
             <div class="col-sm-12 m-auto">
                 <!-- Page Title -->
-                <h1>Contact Us</h1>
+                <h1>Get Touch With Us</h1>
+                <div  style="background-color: red; height: 4px; width: 55px; margin: auto;" class="line">
+
+                </div>
                 <!-- Page Description -->
-                <p> Please contact us with any questions regarding our irrigation system. We're here to help via email, chat, or text.</p>
+                <p style="margin-top: 25px;"> Need to speak to a leading bespoke software development company? We will respond to all your enquires sent to the email address below and via our contact us form within 1 working day </p>
```

Comment:
The content needs to change according to Elzian agro that provide smart agronomy solution.